hack/linter.sh

```diff
@@ -14,22 +14,21 @@ if ! [ -x "$(command -v golangci-lint)" ]; then
 	${DIR}/install_golint.sh -b $GOPATH/bin v1.20.0
 fi
 
-golangci-lint run \
+GO111MODULE=on golangci-lint run \
 	--no-config \
     --disable-all \
 	-E misspell \
 	-E unconvert \
-    -E deadcode \
     -E unconvert \
-    -E gosec \
```

Comment:
:D also this one

Reply:
this is temporary. I needed to remove this, since it would not work for me locally. Need to double check on the reason for deadcode, but the gosec issue was an issue with random number generator which was a false positive for all I can tell.

Reply:
See also https://github.com/jenkins-x/jx/pull/6286/commits/ba9c23b4d7d7c55edd705b8ad44da488b2992d7e